ADAL/src/cache/ADTokenCacheItem.m

```diff
@@ -141,6 +160,9 @@ -(id) initWithCoder:(NSCoder *)aDecoder
         self.refreshToken = [aDecoder decodeObjectOfClass:[NSString class] forKey:@"refreshToken"];
         self.expiresOn = [aDecoder decodeObjectOfClass:[NSDate class] forKey:@"expiresOn"];
         self.userInformation = [aDecoder decodeObjectOfClass:[ADUserInformation class] forKey:@"userInformation"];
+        self.tombstone = [aDecoder decodeBoolForKey:@"tombstone"];
```

Comment:
Here's an idea, have tombstone be an NSDictionary instead and put all of the various fields we only care about if it's a tombstone (correlation ID, bundle ID, error details) in there. That'll reduce the risk on things getting lost between version changes.